src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
```

Comment:
Using @Data annotation in combination with @Setter, @Getter, @ToString, @Builder, @NoArgsConstructor, @AllArgsConstructor can lead to unnecessary code generation and potential issues with code readability and maintainability. It is recommended to use only the necessary annotations based on the actual requirements of the class.